### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by cnizzardini**
Proposing Swagger Bake

https://github.com/cnizzardini/cakephp-swagger-bake

### CODE REVIEW

**Comment by jadb** [comment]
Haven't used myself but looks pretty promising.

File: README.md [hunk 1]
```diff
@@ -339,6 +339,7 @@ Additional lists you might find useful:
 - [Cors plugin](https://github.com/snelg/cakephp-cors) - A lightweight plugin for adding CORS headers to specified endpoints.
 - [CrudJsonApi plugin](https://github.com/FriendsOfCake/crud-json-api) - Crud listener for building [JSON API](https://jsonapi.org/) compliant APIs.
 - [FractalTransformerView plugin](https://github.com/andrej-griniuk/cakephp-fractal-transformer-view) - A plugin which allows using [Fractal transformers](https://fractal.thephpleague.com/transformers/) for your API output.
+- :strawberry: [SwaggerBake plugin](https://github.com/cnizzardini/cakephp-swagger-bake) - This plugin automatically builds your Swagger UI (v3.25) from your existing cake models and routes. A redoc option is also available.
```
Comment (dereuromark) [inline README.md:342]: specifics and version numbers get outdated. I would prefer this to be kept agnostic and instead put that detail into plugin readme etc.

Comment (dereuromark) [reply README.md:342]: cake => can be removed
This plugin => can be removed: Builds ...

Comment (cnizzardini) [reply README.md]: I removed swagger UI version number reference and redundant mention of cake.

**Pull request merged**